config/mappings/sysObjectID.yaml

```diff
@@ -2638,4 +2638,7 @@
 .1.3.6.1.4.1.9148.1.3.1: "Acme Packet 3800"
 .1.3.6.1.4.1.9148.1.3.2: "Acme Packet 3820"
 .1.3.6.1.4.1.9148.1.5: "Acme Packet 6000"
-.1.3.6.1.4.1.9148.1.5.1: "Acme Packet 6300"
\ No newline at end of file
+.1.3.6.1.4.1.9148.1.5.1: "Acme Packet 6300"
+# procurve
+.1.3.6.1.4.1.11.2.3.7.11.154: "HP J9728A 2920-48G"
+
```

Comment:
Since we can read the ProCurve models via the SysDescr, we can remove the mapping.
```suggestion
```